### ISSUE TITLE
UIItemList.ScrollList needs getFocusedIndex()

### ISSUE THREAD

**Issue by ssilverman** [heronarts/LXStudio#7]
UIItemList.ScrollList getFocusedIndex() will tell me which item is currently selected, and also what the "next" item is. Without it, I have to do a manual search to find out what is "next", and it's not reliable if there are ever non-consecutive duplicates.

**Comment by mcslee** [comment]
FYI - note that there is getFocusedItem():
http://lx.studio/api/heronarts/p3lx/ui/component/UIItemList.html#getFocusedItem--

**Comment by mcslee** [comment]
https://github.com/heronarts/P3LX/commit/398a679aed57754e6e4d10d72212f87226a5011f

**Comment by ssilverman** [comment]
getFocusedItem() is still being used to find the index, which is what I actually need. Also, if the list contains “A,B, C, B”, and the selected item is “B”, which index is selected?

**Comment by mcslee** [comment]
Hrm, that's a weird case. Technically the same list item object should not be added twice to a list. You should have two different "B"s in that case - but maybe it'll work fine.

In any case, the commit referenced above adds the methods you'll need.

**Comment by ssilverman** [comment]
Curious, why not? Are they meant to be ordered sets and not lists?

Thank you for these changes! :)

**Comment by ssilverman** [comment]
Right, they are different *Item* objects, but they may have the same labels. To do proper compare, the Item implementations will have to remember to implement `equals()`.
